sdk/containerregistry/azure-containerregistry/azure/containerregistry/_exchange_client.py

```diff
@@ -74,7 +73,7 @@ def get_refresh_token(self, service, **kwargs):
     def exchange_aad_token_for_refresh_token(self, service=None, **kwargs):
         # type: (str, Dict[str, Any]) -> str
         refresh_token = self._client.authentication.exchange_aad_access_token_for_acr_refresh_token(
-            service=service, access_token=self._credential.get_token(self.credential_scope).token, **kwargs
+            service=service, access_token=self._credential.get_token(*self.credential_scopes).token, **kwargs
```

Comment:
Is `credential_scopes` an iterable (the name implies that it is)? If so, does the keyword paramtype need to be updated from `str` to something like `List[str]`?

Reply:
Yes, it should be a `List[str]`.